### ISSUE TITLE
ENV email and password from docker-compose.yml

### ISSUE THREAD

**Issue by freexmonster** [yohamta/dagu#352]
Hello how to pass ENV  from docker compose to baseConfig

Basic Auth in admin.yaml work fine:
```
isBasicAuth: true             # enables basic auth
basicAuthUsername: ${DAGU_USER}       # basic auth user
basicAuthPassword: ${DAGU_USER_PASSWORD}       # basic auth password
```

SMTP server settings in config.yaml  not work
```
smtp:
  host: "smtp.mymail.com"
  port: "587"
  username: ${EMAIL_SERVICE}
  password: ${EMAIL_SERVICE_PASSWORD}
```

**Comment by yohamta** [comment]
Hello, currently it does not expand the env for SMTP server settings. Are you interested in making PR?
https://github.com/yohamta/dagu/blob/main/internal/dag/builder.go#L447-L448

**Comment by yohamta** [comment]
Fixed in v1.9.1. Thanks!
